examples/powershell/vmware-cloud-ngw-ms-teams/handler/script.ps1

```diff
@@ -0,0 +1,92 @@
+
+# Process function Secrets passed in
+$SECRETS_FILE = "/var/openfaas/secrets/ngw-teams-config"
+$SECRETS_CONFIG = (Get-Content -Raw -Path $SECRETS_FILE | ConvertFrom-Json)
+
+# Process payload sent from NGW
+
+$json = $args | ConvertFrom-Json
+if($env:function_debug -eq "true") {
+    Write-Host "DEBUG: json=`"$($json | Format-List | Out-String)`""
+}
+
+if( ($json.event_id -eq "SDDC-PROVISION") -or ($json.event_id -eq "SDDC-DELETE") ) {
```

Comment:
an additional default/else clause with print would be neat to see that the fn run but did not send message due to not matching event_id

Reply:
I was surprised to see VMC NGW event naming not following the general convention, e.g. `"SDDC-PROVISION"` instead of `"SDDCProvisionedEvent"`

Reply:
Yea, sadly there's no consistency nor guidelines on formatting. Its something I've shared w/PM and hopefully we can drive this in a future update of the platform given this feature is still in "Preview"